apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.js

```diff
@@ -19,7 +19,6 @@ function activateSentry() {
 		// of transactions for performance monitoring.
 		// We recommend adjusting this value in production
 		release: 'wpcom-test-01',
```

Comment:
I think there is an import we need to remove on line 2 in this file, and then the integration on line 16

Reply:
Oh, true! Thanks for catching that.